loadtest/control/simulcontroller/actions.go

```diff
@@ -2062,3 +2062,37 @@ func (c *SimulController) openPermalink(u user.User) control.UserActionResponse
 
 	return control.UserActionResponse{Info: fmt.Sprintf("clicked permalink on post %s", postID)}
 }
+
+func (c *SimulController) generateUserReport(u user.User) control.UserActionResponse {
+	// Simulate scrolling through the entire list of users
+	// (should be similar to generating the complete report and exporting it)
+
+	lastColumnValue := ""
+	lastId := ""
+	totalUsers := 0
+
+	for {
```

Comment:
Is there a way to throttle this a little bit? Or does it make sense to hit the API as fast as possible?

Reply:
> Is there a way to throttle this a little bit? Or does it make sense to hit the API as fast as possible?

What we want is to simulate what real user would do. Again, we don't have data on that yet, but I guess we can do an educated guess and try to throttle it to make it more like a human-triggered action.